File: patterns/c-patterns/pattern37.c
// Pattern 37: ...

// 1
// **
// 123
// ****
// 12345

#include <stdio.h>

int main()
{
    int i, j;
    int n = 5;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= i; j++)
        {
            if (i % 2 == 0)
                printf("*");
            else
                printf("%d", j);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern37.js
// Pattern 37: ...

// 1
// **
// 123
// ****
// 12345

const n = 5;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = 1; j <= i; j++) {
        if (i % 2 === 0) {
            row += '*';
        } else {
            row += j;
        }
    }

    console.log(row);
}